tools/calculator/calculator.html

```diff
@@ -0,0 +1,422 @@
+            background-color: #2c7873;
+            color: white;
+            border-radius: 50%;
+            font-size: 14px;
+            font-weight: bold;
+        }
+        .tooltip-text {
+            visibility: hidden;
+            width: 300px;
+            background-color: #555;
+            color: #fff;
+            text-align: left;
+            border-radius: 6px;
+            padding: 8px;
+            position: absolute;
+            z-index: 1;
+            left: 125%;
+            top: 50%;
+            transform: translateY(-50%);
+            opacity: 0;
+            transition: opacity 0.3s;
+            font-weight: normal;
+            font-size: 14px;
+            line-height: 1.4;
+        }
+        .tooltip:hover .tooltip-text {
+            visibility: visible;
+            opacity: 0.95;
+        }
+        .results {
+            margin-top: 30px;
+            background-color: #e9f7ef;
+            padding: 15px;
+            border-radius: 8px;
+        }
+        .result-item {
+            margin-bottom: 10px;
+            font-size: 16px;
+            display: flex;
+            align-items: center;
+        }
+        .value {
+            font-weight: bold;
+            color: #2c7873;
+            margin-right: 10px;
+        }
+        button {
+            background-color: #2c7873;
+            color: white;
+            border: none;
+            padding: 10px 15px;
+            border-radius: 4px;
+            cursor: pointer;
+            font-size: 16px;
+            display: block;
+            margin: 20px auto;
+        }
+        button:hover {
+            background-color: #1f5f5b;
+        }
+        .reset {
+            background-color: #888;
+            margin-left: 10px;
+        }
+        .reset:hover {
+            background-color: #666;
+        }
+        .button-group {
+            display: flex;
+            justify-content: center;
+        }
+    </style>
+</head>
+<body>
+    <h1>EigenDA Resource Calculator</h1>
+    
+    <div class="calculator">
+        <div class="input-group">
+            <label for="maxThroughput">Max Throughput (MB/s):</label>
+            <input type="number" id="maxThroughput" value="50" step="0.1" min="0">
+            <div class="tooltip">
+                <div class="tooltip-icon">?</div>
+                <span class="tooltip-text">The maximum throughput supported by EigenDA.</span>
+            </div>
+        </div>
+        
+        <div class="input-group">
+            <label for="sumOfStake">Sum of Stake Across All Quorums:</label>
+            <input type="number" id="sumOfStake" value="0.125" step="0.001" min="0" max="1">
+            <div class="tooltip">
+                <div class="tooltip-icon">?</div>
+                <span class="tooltip-text">The sum of an operator's stake percentage across all quorums. For example, if an operator has 10% stake in 3 quorums, the sum of stake across all quorums is 30%.</span>
```

Comment:
This mentions percentage, but the field actually accepts a fraction (0–1)

Reply:
```
<span class="tooltip-text">The sum of an operator's stake across all quorums. 1.0 means 100% stake for one quorum, 2.0 means 100% stake across two quorums. For example, if an operator has 10% stake in 3 quorums, the sum of stake across all quorums is 30%, and so you'd use 0.3 for field.</span>
```